website/docs/reference/global-configs/resource-type.md

```diff
@@ -42,7 +42,7 @@ The available resource types are:
 
 </VersionBlock>
 
-<VersionBlock firstVersion="1.8">
+<VersionBlock lastVersion="1.8">
```

Comment:
Should we keep the firstversion rule in since we still display v1.6 and v1.7?
```suggestion
<VersionBlock firstVersion="1.8" lastVersion="1.8">
```